packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -47,6 +48,7 @@ import {
   useTableSelectAllCheckbox,
   useTableSelectionCheckbox
 } from '@react-aria/table';
+import {useTableColumnResizeState} from './useTableColumnResizeState';
```

Comment:
got two different imports from `useTableColumnResizeState` here, can combine the two